crates/openvino/tests/detect-inception.rs

```diff
@@ -56,14 +92,15 @@ fn detect_inception() {
         .collect();
     results.sort_by(|a, b| b.1.partial_cmp(&a.1).unwrap());
 
+    //Result buffer seem off by 1
```

Comment:
What is in `results[0]`?

Reply:
Removed test as it was outdated - https://github.com/intel/openvino-rs/issues/93